docs/selectors.md

```diff
@@ -47,6 +47,11 @@ document
 
 If a selector needs to include `>>` in the body, it should be escaped inside a string to not be confused with chaining separator, e.g. `text="some >> text"`.
 
+### Indexed matches
+By default, selectors resolve to the first matching element, according to the DOM traversal order. A selector can be followed by `{n}` to match the n-th element instead.
```

Comment:
```suggestion
By default, selectors resolve to the first matching element, according to the DOM traversal order. A Playwright selector can be followed by `{n}` to match the n-th element instead.
```

No meaning change here, but adding the word playwright helps people jumping to this part of the docs without reading the above sections.